test/redbird_test.exs

```diff
@@ -106,9 +93,12 @@ defmodule RedbirdTest do
   end
 
   describe "delete" do
+    # TODO: Does running this call the key preparer function?
+    # - No it doesn't
+    # - So this shows that there are two ways to use the library. you can set directly or indirectly
```

Comment:
😄  I like the back and forth you went through here. Is this dual use something that we should document somehow?

Reply:
Sooo embarrassing, leaving my comments all over the place. 😊